packages/salesforcedx-vscode-lwc/package.json

```diff
@@ -357,6 +357,39 @@
         "title": "%force_lightning_lwc_test_stop_watching_all_tests_text%"
       }
     ],
+    "debuggers": [
+      {
+        "type": "vscode-lwc-tests",
+        "label": "%force_lightning_lwc_debugger%",
+        "languages": [
+          "javascript",
+          "typescript"
```

Comment:
Do we support LWCs running with typescript?

Reply:
good call...doesn't look like it's a thing currently. It can be done but it's not going to be an out of the box feature.